media-proxy/src/mesh/conn_rdma_tx.cc

```diff
@@ -0,0 +1,186 @@
+#include "conn_rdma_tx.h"
+#include <stdexcept>
+#include <queue>
+
+namespace mesh {
+
+namespace connection {
+
+RdmaTx::RdmaTx() : Rdma() { init_buf_available(); }
```

Comment:
You dont have to call ```Rdma()```

Reply:
fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/286/commits/1e7f66891946f5a904408fb568ca119b7e04eaa0